sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_requests.py

```diff
@@ -7,12 +7,18 @@
 import hashlib
 import base64
 import hmac
+from yarl import URL
+
 from azure.core.pipeline.policies import HTTPPolicy
-from ._utils import get_current_utc_time
+from azure.core.pipeline.transport import AioHttpTransport  # pylint:disable=non-abstract-transport-import,no-name-in-module
```

Comment:
Why we move it here?

Reply:
Just moved imports as others to the start of each file. Any reason to treat them differently?

Reply:
The problem is if aiohttp is not installed, it fails the sync client which is not what we want.

Reply:
Got it